CHANGELOG.md

```diff
@@ -4,22 +4,68 @@ All notable changes to this project will be documented in this file.
+ - The new `pyln-testing` package now contains the testing infrastructure so it can be reused to test against c-lighting in external projects ([3218](https://github.com/ElementsProject/lightning/pull/3218))
+ - config: configuration files now support `include`. ([3268](https://github.com/ElementsProject/lightning/pull/3268))
+ - options: Allow the Tor inbound service port differ from 9735 ([3155](https://github.com/ElementsProject/lightning/pull/3155))
+ - options: Persistent Tor address support ([3155](https://github.com/ElementsProject/lightning/pull/3155))
+ - plugins: A new plugin hook, `rpc_command` allows a plugin to take over `lightningd` for any RPC command. ([2925](https://github.com/ElementsProject/lightning/pull/2925))
+ - plugins: Allow the `accepter` to specify an upfront_shutdown_script for a channel via a `close_to` field in the openchannel hook result ([3280](https://github.com/ElementsProject/lightning/pull/3280))
+ - plugins: Plugins may now handle modern TLV-style payloads via the `htlc_accepted` hook ([3260](https://github.com/ElementsProject/lightning/pull/3260))
+ - plugins: libplugin now supports writing plugins which register to hooks ([3317](https://github.com/ElementsProject/lightning/pull/3317))
+ - plugins: libplugin now supports writing plugins which register to notifications ([3317](https://github.com/ElementsProject/lightning/pull/3317))
+ - protocol: Payment amount fuzzing is restored, but through shadow routing. ([3212](https://github.com/ElementsProject/lightning/pull/3212))
+ - protocol: We now signal the network we are running on at init. ([3300](https://github.com/ElementsProject/lightning/pull/3300))
+ - quirks: Workaround LND's `reply_channel_range` issues instead of sending error. ([3264](https://github.com/ElementsProject/lightning/pull/3264))
+ - tools: A new command, `guesstoremote`, is added to the hsmtool. It is meant to be used to recover funds after an unilateral close of a channel with `option_static_remotekey` enabled. ([3292](https://github.com/ElementsProject/lightning/pull/3292))
+
 ### Changed
 
+:warning: The default network and the default location of the lightning home directory changed. Please make sure that the configuration, key file and database are moved into the network-specific subdirectory. :warning:
+
+ - config: Default network (new installs) is now bitcoin, not testnet. ([3268](https://github.com/ElementsProject/lightning/pull/3268))
+ - config: Lightning directory, plugins and files moved into `<network>/` subdir ([3268](https://github.com/ElementsProject/lightning/pull/3268))
+ - JSON API: The `fundchannel` command now tries to connect to the peer before funding the channel, no need to `connect` before `fundchannel` anymore ! ([3314](https://github.com/ElementsProject/lightning/pull/3314))
```

Comment:
this isn't technically correct -- you still need to call connect with the ip address and port except for nodes that are included in the seed nodes (i think, i'm not 100% sure) cc @darosior for clarification

Reply:
Added a short clarification that an address needs to be known.